cache.js

```diff
@@ -0,0 +1,8 @@
+const redis = require('redis');
+const client = redis.createClient();
```

Comment:
Ideally we would be running a redis cluster in docker, this was just for demonstration purposes.